zglfw + zgui macos builds failing

Any sample with both zglfw and zgui fails to build for both x86_64 and aarch64 macos targets.

minimal_zgpu_zgui,
minimal_zgui_glfw_gl,
physically_based_rendering_wgpu,

zig version 0.13.0-dev.351+64ef45eb0

```zig build minimal_zgpu_zgui -Dtarget=x86_64-macos```
```zig build minimal_zgpu_zgui -Dtarget=aarch64-macos```

```minimal_zgui_glfw_gl-run
└─ run minimal_zgui_glfw_gl
   └─ zig build-exe minimal_zgui_glfw_gl Debug x86_64-macos
      └─ zig build-lib imgui Debug x86_64-macos 1 errors
/Users/connor/git/zig-gamedev/libs/zglfw/libs/glfw/include/GLFW/glfw3native.h:112:13: error: 'ApplicationServices/ApplicationServices.h' file not found
   #include <ApplicationServices/ApplicationServices.h>
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/connor/git/zig-gamedev/libs/zgui/libs/imgui/backends/imgui_impl_glfw.cpp:101:10: note: in file included from /Users/connor/git/zig-gamedev/libs/zgui/libs/imgui/backends/imgui_impl_glfw.cpp:101:
#include <GLFW/glfw3native.h>   // for glfwGetCocoaWindow()
         ^
error: warning(compilation): failed to delete '/Users/connor/git/zig-gamedev/.zig-cache/tmp/9a2bd6786cc7dd3d-imgui_impl_glfw.o.d': FileNotFound

error: the following command failed with 1 compilation errors:
/Users/connor/zig/0.13.0-dev.351+64ef45eb0/files/zig build-lib -cflags -fno-sanitize=undefined -- /Users/connor/git/zig-gamedev/libs/zgui/src/zgui.cpp -cflags -fno-sanitize=undefined -- /Users/connor/git/zig-gamedev/libs/zgui/libs/imgui/imgui.cpp /Users/connor/git/zig-gamedev/libs/zgui/libs/imgui/imgui_widgets.cpp /Users/connor/git/zig-gamedev/libs/zgui/libs/imgui/imgui_tables.cpp /Users/connor/git/zig-gamedev/libs/zgui/libs/imgui/imgui_draw.cpp /Users/connor/git/zig-gamedev/libs/zgui/libs/imgui/imgui_demo.cpp -cflags -fno-sanitize=undefined -- /Users/connor/git/zig-gamedev/libs/zgui/libs/imgui/implot_demo.cpp /Users/connor/git/zig-gamedev/libs/zgui/libs/imgui/implot.cpp /Users/connor/git/zig-gamedev/libs/zgui/libs/imgui/implot_items.cpp -cflags -fno-sanitize=undefined -DIMGUI_IMPL_OPENGL_LOADER_CUSTOM -- /Users/connor/git/zig-gamedev/libs/zgui/libs/imgui/backends/imgui_impl_glfw.cpp /Users/connor/git/zig-gamedev/libs/zgui/libs/imgui/backends/imgui_impl_opengl3.cpp -ODebug -target x86_64-macos -mcpu baseline -I /Users/connor/git/zig-gamedev/libs/zgui/libs -I /Users/connor/git/zig-gamedev/libs/zgui/libs/imgui -I /Users/connor/git/zig-gamedev/libs/zglfw/libs/glfw/include -DZGUI_IMPLOT=1 -DZGUI_TE=0 -Mroot -lc++ -lc --cache-dir /Users/connor/git/zig-gamedev/.zig-cache --global-cache-dir /Users/connor/.cache/zig --name imgui -static --listen=-
Build Summary: 7/12 steps succeeded; 1 failed (disable with --summary none)
minimal_zgui_glfw_gl-run transitive failure
└─ run minimal_zgui_glfw_gl transitive failure
   ├─ zig build-exe minimal_zgui_glfw_gl Debug x86_64-macos transitive failure
   │  └─ zig build-lib imgui Debug x86_64-macos 1 errors
   └─ install minimal_zgui_glfw_gl transitive failure
      └─ zig build-exe minimal_zgui_glfw_gl Debug x86_64-macos (+8 more reused dependencies)
error: the following build command failed with exit code 1:
/Users/connor/git/zig-gamedev/.zig-cache/o/7dcc050882c226c4bc55250be91dc279/build /Users/connor/zig/0.13.0-dev.351+64ef45eb0/files/zig /Users/connor/git/zig-gamedev /Users/connor/git/zig-gamedev/.zig-cache /Users/connor/.cache/zig --seed 0x31e903b -Zfd1b67ccade83f69 minimal_zgui_glfw_gl-run -Dtarget=x86_64-macos
```

--- a/libs/zgui/build.zig
+++ b/libs/zgui/build.zig
@@ -51,7 +51,7 @@ pub fn build(b: *std.Build) void {
         },
     });
 
-    const cflags = &.{"-fno-sanitize=undefined"};
+    const cflags = &.{ "-fno-sanitize=undefined", "-Wno-elaborated-enum-base" };
 
     const imgui = if (options.shared) blk: {
         const lib = b.addSharedLibrary(.{
@@ -247,6 +247,12 @@ pub fn build(b: *std.Build) void {
             });
         },
         .no_backend => {},
+    }
+
+    if (target.result.os.tag == .macos) {
+        const system_sdk = b.dependency("system_sdk", .{});
+        imgui.addSystemIncludePath(system_sdk.path("macos12/usr/include"));
+        imgui.addFrameworkPath(system_sdk.path("macos12/System/Library/Frameworks"));
     }
 
     const test_step = b.step("test", "Run zgui tests");

--- a/libs/zgui/build.zig.zon
+++ b/libs/zgui/build.zig.zon
@@ -11,5 +11,6 @@
     .dependencies = .{
         .zglfw = .{ .path = "../zglfw" },
         .zgpu = .{ .path = "../zgpu" },
+        .system_sdk = .{ .path = "../system-sdk" },
     },
 }

